css/selectors/-moz-list-bullet.json

```
@@ -12,7 +12,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "≤72"
+              "version_added": "≤69"
```

See: https://github.com/caugner/firefox-releases/blame/v1/content/shared/public/nsCSSPseudoElementList.h#L76

```suggestion
              "version_added": "1"
```